bittensor/subtensor.py

```diff
@@ -3420,24 +3506,35 @@ def get_axon_info(
                 hotkey=hotkey_ss58,
                 coldkey="",
             )
-
         return None
 
+    # TODO: check if someone still use this method. bittensor not.
```

Comment:
`get_prometheus_info` is not used by any subnets in finney
But it is used in subtensor in 
`neuron_info`,  and `serving`